unittests/misc_tests.cpp

```diff
@@ -1156,7 +1156,7 @@ BOOST_AUTO_TEST_CASE(stable_priority_queue_test) {
 
      appbase::execution_priority_queue pri_queue;
      auto io_serv = std::make_shared<boost::asio::io_context>();
-     auto work_ptr = std::make_unique<boost::asio::io_context::work>(*io_serv);
+     auto work_ptr = boost::asio::make_work_guard(*io_serv);
```

Comment:
Nit: not a `_ptr` type anymore.

Reply:
https://github.com/AntelopeIO/spring/pull/1384/commits/fa833316518b9de43c5a257803bd67be788b2a35